### DISCUSSION TITLE
how-to-notify-services-when-post-jekyll/

### DISCUSSION THREAD

**Discussion by giscus** [natelandau/natelandau.com#83] [Comments]
# how-to-notify-services-when-post-jekyll/

Use a rakefile to ping PingOMatic, Pubsubhubbub, Google, and Bing when you post new content to your Jekyll site.

https://natelandau.com/how-to-notify-services-when-post-jekyll/?utm_source=RSS&utm_medium=RSS&utm_campaign=HowtonotifywebserviceswhenyouposttoJekyll

**Comment by h2a2d0410** [comment]
<?xml version="1.0" encoding="UTF-8"?>
<urlset xmlns="http://sitemaps.org">
  
  
  <url>
    <loc>https://yourdomain.com</loc>
    <lastmod>2026-07-21</lastmod>
    <changefreq>daily</changefreq>
    <priority>1.0</priority>
  </url>

  
  <url>
    <loc>https://yourdomain.com%D8%AE%D8%B1%D9%8A%D8%B7%D8%A9-%D8%A7%D9%84%D9%85%D9%88%D9%82%D8%B9</loc>
    <lastmod>2026-07-20</lastmod>
    <changefreq>monthly</changefreq>
    <priority>0.8</priority>
  </url>

</urlset>
